sorc/emcsfc_snow2mdl.fd/snowdat.f

```diff
@@ -96,41 +96,35 @@ module snowdat
                        9*0,255,180*0/
  data kgds_afwa_sh_8th/5,2*512,20826,-125000,8,-80000,2*47625,128, &
                        9*0,255,180*0/
-
  contains
-
+!>
```

Comment:
Make this match the example.

!> Some one-sentence description.
!!
!! More sentences of desciption. 

Take out the name of the subprogram, the word "abstact:", captialize the sentences.